src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/alignment/AssemblyContigAlignmentsConfigPicker.java

```diff
@@ -460,12 +595,41 @@ private static AssemblyContigWithFineTunedAlignments updateContigMappingsWithGap
         final Comparator<AssemblyContigWithFineTunedAlignments> mismatchSecond
                 = (AssemblyContigWithFineTunedAlignments x, AssemblyContigWithFineTunedAlignments y)
                 -> Integer.compare(x.getAlignments().stream().mapToInt(ai -> ai.mismatches).sum(),
-                                   y.getAlignments().stream().mapToInt(ai -> ai.mismatches).sum());
+                y.getAlignments().stream().mapToInt(ai -> ai.mismatches).sum());
         return numFirst.thenComparing(mismatchSecond);
     }
 
+    // functionality group 3.1: split gaps
+
+    /**
+     * Split the good alignments stored in {@code configuration}.
+     *
+     * Note that an edge case is possible where for some gapped alignment,
+     * after gap-split, one or more child alignments might be contained by
+     * another original alignment.
+     *
+     * Example:
+     * picture: ref span by gapped original alignment   -------------          --------
+     *          ref span by overlapping other alignment                  ---------------------------
+     *
+     * TODO:
+     * Here we have two options:
+     *  1) keep doing the gap split, and compare if the gapped original alignment offers
+     *     better read coverage/breadth compared to the other, if so, keep the children alignments from the gap split,
+     *     and ditch the other alignment, otherwise ditch the alignment gapped original alignment,
+     *     the aim is to keep the split children alignments together
+     *  2) drop the gap-split children alignment whose read span is contained in other's.
```

Comment:
"drop the gap-split child alignments whose read spans are contained in other alignment spans"